LIMS_IMAGE/web/src/settings.py

```diff
@@ -120,7 +120,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'PST8PDT'
```

Comment:
maybe add a comment that this is PST with daylight saving accounted for?

Reply:
Good catch, I'll add that